scripts/core/exp-bindless-images.yml

```diff
@@ -86,6 +86,24 @@ etors:
     - name: INTEROP_SEMAPHORE_EXPORT_SUPPORT_EXP
       value: "0x200F"
       desc: "[$x_bool_t] returns true if the device supports exporting internal event resources"
+    - name: BINDLESS_SAMPLED_IMAGE_FETCH_1D_USM
```

Comment:
Since bindless is still experimental, I'm wondering if these enums should have the `EXP` suffix?

Reply:
Good catch, I missed that. Updated to have `_EXP` suffix.